requirements.txt

```
@@ -1,7 +1,13 @@
-f90nml>=1.1.0
-appdirs>=1.4.0
-requests>=2.22.0
-pyyaml>=5.0
-gcsfs>=0.7.0
-fsspec>=0.8.0
-dacite>=1.6.0
+pip
+bump2version
+wheel
+flake8
```

We want to keep the pins on black and flake8 because version changes can change the linting requirements. Here probably makes more sense than in the circleci yaml.

```suggestion
flake8==3.7.8
```